nx-X11/lib/X11/XlibInt.c

```diff
@@ -3696,9 +4113,13 @@ _XData32(
  *       and so, you may be better off using gethostname (if it exists).
  */
 
-#if (defined(_POSIX_SOURCE) && !defined(AIXV3) && !defined(__QNX__)) || defined(hpux) || defined(USG) || defined(SVR4)
```

Comment:
What is the USG definition? If we remove it here, shouldn't we remove it all over the nx-libs source tree?

Personally, I'd say we leave it in place here and do another clean-up run that removed USG support everywhere.